specification/monitor/resource-manager/readme.md

```diff
@@ -34,7 +34,38 @@ title: MonitorManagementClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-2021-04
+```
+
+### Tag: package-2021-04
+
+These settings apply only when `--tag=package-2021-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-04'
+input-file:
+- Microsoft.Insights/stable/2015-04-01/autoscale_API.json
+- Microsoft.Insights/stable/2015-04-01/operations_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRulesIncidents_API.json
+- Microsoft.Insights/stable/2016-03-01/alertRules_API.json
+- Microsoft.Insights/stable/2016-03-01/logProfiles_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettings_API.json
+- Microsoft.Insights/preview/2017-05-01-preview/diagnosticsSettingsCategories_API.json
+- Microsoft.Insights/stable/2019-06-01/actionGroups_API.json
+- Microsoft.Insights/stable/2015-04-01/activityLogs_API.json
```

Comment:
Per request here: https://github.com/Azure/azure-sdk-for-go/issues/14704
Could this be updated to a newer api-version?

Reply:
The only spec in [2017-03-01-preview](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/monitor/resource-manager/Microsoft.Insights/preview/2017-03-01-preview) is activityLogAlerts_API.json.

The latest for activityLogs_API.json is 2015-04-01.
https://docs.microsoft.com/en-us/rest/api/monitor/activitylogs

I don't see other version of Activity Logs.
Or, Am I misunderstanding?

Reply:
Sure, I am fine with this. Please take a look at the comment below